controllers/sms-gateway.js

```diff
@@ -119,6 +120,7 @@ module.exports = {
       .then(JSON.parse)
       .then(function(request) {
         // Process webapp-terminating messages asynchronously
+        // TODO these promises do nothing!
```

Comment:
👍 Glad that's covered.  I also could have done a better job documenting the intention of this in the first place.